packages/cli/src/error_handler.ts

```diff
@@ -13,6 +16,13 @@ type HandleErrorProps = {
   command?: string;
 };
 
+/**
+ * Reset the hasError flag
+ */
+export const resetHasError = () => {
+  hasError = false;
```

Comment:
Instead of using the `resetHasError`, we should mock the `hasError` and reset the mock. Do something like this https://stackoverflow.com/questions/62694921/how-to-mock-variable-in-jest

Reply:
I'm not sure that will work since ES6 imports are immutable. But I'm willing to be proven wrong.